### DISCUSSION TITLE
How could I get webpack version in plugin

### DISCUSSION THREAD

**Discussion by yangguansen** [webpack/webpack#15526] [Q&A] [answered]
I'm making a webpack plugin, for compatible webpack@4 and webpack@5, how could i get webpack version in `apply()`?

**Comment by vankop** [comment, marked as answer]
in webpack@5 compiler has `webpack` property (`compiler.webpack`). Also if you want full version in webpack@5 `compiler.webpack.version`

**Reply by yangguansen** [reply]
So how could i get flag of webpack@4?

**Reply by vankop** [reply]
```js
const isWebpack5 = "webpack" in compiler
```
otherwise it is webpack@4
